crates/ui/src/components/web_view.rs

```diff
@@ -0,0 +1,49 @@
+struct WebViewTest {
+    views: Vec<Arc<WebView>>,
+}
+
+impl WebViewTest {
+    fn new(num_views: usize, handle: &dyn HasWindowHandle) -> Self {
```

Comment:
maybe i will make this an element? i also may just leave it only as a view in the outputs.rs file.